Makefile

```diff
@@ -275,13 +275,13 @@ clean: clean-cross-build
 .PHONY: clean
 
 licensecheck: microshift bin/lichen
-	bin/lichen -c .lichen.yaml microshift
+	$(GO_BUILD_BINDIR)/lichen -c .lichen.yaml $(GO_BUILD_BINDIR)/microshift
 
 bin:
 	mkdir -p $@
 
 bin/lichen: bin vendor/modules.txt
-	GOBIN=$(realpath ./bin) go install github.com/uw-labs/lichen@latest
+	GOBIN=$(realpath $(GO_BUILD_BINDIR)) go install github.com/uw-labs/lichen@latest
```

Comment:
Let's not use ./bin path. The tool should be installed under _output instead.

Reply:
Yeah, that was the issue, the ./bin dir was at the root, that `GO_BUILD_BINDIR` var points to `_output/bin`, moved it over to the script for this installation step.